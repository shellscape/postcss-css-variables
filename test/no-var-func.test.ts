import { fixture, process } from './helpers';

const tests = {
  'should work when there are no var() functions to consume declarations': 'no-var-func',
  'should work when there are no var() functions(just `:root`) to consume declarations':
    'no-var-func-just-root'
};

describe('no-var-func', async () => {
  test.each(Object.entries(tests))('%s', async (_desc, file) => {
    const css = await fixture(`no-var-func/${file}`);
    const result = await process(css);
    expect(result).toMatchSnapshot();
  });
});
