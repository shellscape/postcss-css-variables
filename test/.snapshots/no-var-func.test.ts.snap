// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`no-var-func > should work when there are no var() functions to consume declarations 1`] = `".box{width:100px}"`;

exports[`no-var-func > should work when there are no var() functions(just \`:root\`) to consume declarations 1`] = `""`;
